source/docs/articles/sites/code/unsupported-modules-plugins.md

```diff
@@ -6,56 +6,92 @@ keywords: modules, plugins, unsupported, drupal, wordpress
 This article lists modules and plugins that do not work with or are currently unsupported on the Pantheon platform.
 We do not prevent you from installing and using these plugins/modules; however, they will not work as expected and we cannot provide troubleshooting support.
 
-##Drupal Modules
-
-- APC
+## Drupal Modules
+<hr>
+### APC
+**Issue**: APC lacks the ability to span multiple server environments.
+
+**Solution**: Pantheon recommends Redis as a caching backend, which has better performance.
+<hr>
+### Adaptive Image Styles
+**Issue**: This module requires edits to the `nginx.conf` which is not currently supported on the platform. See [Known Limitations](/docs/articles/sites/known-limitations/#nginx.conf).
+<hr>
+### Apache Solr Search
+**Issue**: If Acquia Solr modules are present in the site codebase (even if disabled) and Pantheon Apache Solr is enabled, the site will be unable to connect to Solr server.
+
+**Solution**: Delete the Acquia Solr modules from the codebase and then disable and re-enable the Pantheon Apache Solr module.
+<hr>
+### Background Process  
+**Issue**: The module allows for Drupal to perform "parallel" (asynchronous non-blocking mode) requests. However, there are a number of limitations working in a distributed environment and function correctly on the platform.
 
-- Adaptive Image Styles
+<hr>
+### Backup & Migrate
+**Issue**: The Backup and Migrate module can create large archives and cause issues with the workflow tools on the dashboard. See [Backup Creation](/docs/articles/sites/backups/backup-creation/#drupal-note-why-is-backup-&-migrate-not-recommended-on-pantheon?).
 
-- Apache Solr Search
-**Issue** If Acquia Solr modules are present in the site codebase (even if disabled) and Pantheon Apache Solr is enabled, the site will be unable to connect to Solr server.  
-**Solution** Delete the Acquia Solr modules from the codebase and then disable and re-enable the Pantheon Apache Solr module.
+**Solution**: You can use the automated backups that are available on the dashboard for each environment.
+<hr>
+### Boost
+**Issue**: We do not recommend the boost module on the platform as Varnish is running. Running Boost in addition to Varnish can cause caching issues.
```

Comment:
suggest:
"Boost is an unnecessary caching layer that may cause issues. Every site on Pantheon can leverage our robust Varnish infrastructure that caches pages for anonymous visitors at the highest possible performance. See [Working with Varnish](/docs/articles/sites/varnish/)"